src/compiler/tsc.ts

```diff
@@ -191,7 +191,7 @@ namespace ts {
             return sys.exit(ExitStatus.Success);
         }
 
-        if (commandLine.options.watch) {
+        if (commandLine.options.watch && commandLine.options.hasOwnProperty('watch')) { // FireFox has Object.prototype.watch

```

Comment:
Or shall I go like this below?
It bugs me that the comment now hangs above the whole options.watch handling if statement, not quite helping the clarity.

        if (commandLine.options.watch
            // Firefox has Object.prototype.watch
            && commandLine.options.hasOwnProperty('watch')) {